admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -158,6 +159,7 @@ you can set the following parameters inside the :file:`config/config.php`.
     'overwriteprotocol' => 'https',
     'overwritewebroot'  => '/domain.tld/nextcloud',
     'overwritecondaddr' => '^10\.0\.0\.1$',
+    'overwrite.cli.url' => 'https://domain.tld/,
```

Comment:
:see_no_evil: 

`'overwrite.cli.url' => 'https://domain.tld/nextcloud/,` was actually correct. 
I merged the suggestion to fast ;)